source/StepBro.Core/Parser/StepBroListener.cs

```diff
@@ -219,53 +219,63 @@ public override void ExitFileVariableSimple([NotNull] SBP.FileVariableSimpleCont
 
         public override void ExitFileVariableWithPropertyBlock([NotNull] SBP.FileVariableWithPropertyBlockContext context)
         {
-            TypeReference type = m_variableType;
-            if (type == null)
+            if (m_variableType == null || m_creatorType == null)
             {
                 return;
             }
+            var t = context.children[4].GetType();
```

Comment:
Not sure what the variable is used for, but could have a name that explains more of what it is used for (Or remove it if it is not used at all) :)

Reply:
That's right. Line 227 has what's needed.